src/ProjectTemplates/Web.ItemTemplates/content/RazorComponent/.template.config/template.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "ASP.NET"
+  ],
+  "name": "Razor Component",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A reusable web UI component implemented with Razor",
```

Comment:
Let's remove "web" in the event we use Razor components for native UI scenarios.